src/NetworkPortInstantiation.php

```diff
@@ -59,13 +59,19 @@ class NetworkPortInstantiation extends CommonDBChild
     public $auto_message_on_action   = false;
 
     // From CommonDBChild
-    public static $itemtype       = 'NetworkPort';
+    /** @var class-string<CommonDBTM> string */
+    public static $itemtype       = NetworkPort::class;
+    /** @var string */
     public static $items_id       = 'networkports_id';
+    /** @var bool */
```

Comment:
Types are inherited from `CommonDBChild`/`CommonDBTM`.

```suggestion
    public static $itemtype       = NetworkPort::class;
    public static $items_id       = 'networkports_id';
```